api/src/config.rs

```diff
@@ -755,6 +756,9 @@ pub struct FileCacheConfig {
     /// Key for data encryption, a heximal representation of [u8; 32].
     #[serde(default)]
     pub encryption_key: String,
+    /// disbale chunk map, it is assumed that all data is ready
+    #[serde(default)]
+    pub disable_chunk_map: bool,
```

Comment:
Is this option similar to `disable_indexed_map` ? And should we export the option to the user?